doc/UseCases/uc-gconf_x-road_use_case_model_for_global_configuration_distribution_1.4_Y-883-8.md

```diff
@@ -0,0 +1,1918 @@
+
+**Component:** Security server
+
+**Actor**: SS administrator
+
+**Brief Description**: SS administrator uploads the configuration anchor
+file to the system. System displays the details of the anchor and asks
+for SS administrator to confirm the upload. After confirmation, the
+system verifies that the configuration downloaded from the source
+pointed by this anchor is usable and starts using the uploaded anchor.
+
+**Preconditions**: SS administrator has received a configuration anchor
+file form the internal configuration provider and validated the
+integrity of the anchor.
+
+**Postconditions**: -
+
+**Trigger**: The configuration anchor needs to be uploaded
+
+-   on system initialization or
+
+-   when the X-Road governing agency has notified SS administrator that
+    the configuration anchor needs to be updated.
+
+**Main Success Scenario**:
+
+1.  SS administrator selects to upload a configuration anchor file.
+
+2.  SS administrator inserts the path to the anchor file in the local
+    file system.
+
+3.  System verifies that the file is a valid configuration anchor file
+    by validating the uploaded file against the configuration anchor
+    schema.
+
+4.  System verifies that the instance identifier in the anchor file
+    corresponds to the instance identifier of the security server.
+
+5.  System calculates and displays the SHA-224 hash value and the
+    generation time of the selected anchor file and prompts for
+    confirmation.
+
+6.  SS administrator confirms.
+
+7.  System verifies that the anchor file is functional by downloading
+    configuration from the source pointed by the anchor: 2.3.4.
+
+8.  System saves the configuration anchor (overwriting the existing
+    anchor if such exists).
+
+9.  System logs the event “Upload configuration anchor” to the audit
+    log.
+
+**Extensions**:
+
+3a. The selected file is not a valid configuration anchor file.
+    - 3a.1. System displays the error message: “Configuration anchor import failed: invalid anchor file”.
+    - 3a.2. SS administrator selects to reinsert the path to the configuration anchor file. Use case continues from step 3.
+        - 3a.2a. SS administrator selects to terminate the use case.
+
+- 4a. The instance identifier in the anchor file does not correspond to the instance identifier of the security server.
+    - 4a.1. System displays the error message: “Configuration anchor upload failed: unexpected instance identifier found in anchor”.
+    - 4a.2. System logs the event “Upload configuration anchor failed” to the audit log.
+    - 4a.3. SS administrator selects to reinsert the path to the configuration anchor file. Use case continues from step 3.
+        - 4a.3a. SS administrator selects to terminate the use case.
+
+- 6a. SS administrator cancels the import.
+    - 6a.2. Use case terminates.
+
+- 7a. Downloading of the internal configuration fails.
+    - 7a.1. System displays the error message: “Configuration source cannot be reac
```

Comment:
Number missing, this causes the content listing to not work and the last number to incorrectly be 2.3.6

Reply:
Not sure what you mean here. first extension 3a is missing ul though. if you mean 6a.2 not being 6a.1 it was already that way in the word doc.